app/src/main/java/com/github/h3lp3rs/h3lp/MapsFragment.kt

```diff
@@ -212,8 +237,10 @@ class MapsFragment : Fragment(), CoroutineScope by MainScope(), GoogleMap.OnPoly
      * Utility function to remove all markers corresponding to one utility
      */
     fun removeMarkers(utility: String) {
-        for (marker in placedMarkers[utility]!!) {
-            marker.remove()
+        if(placedMarkers.containsKey(utility)){
+            for (marker in placedMarkers[utility]!!) {
+                marker.remove()
+            }
```

Comment:
Making this a little prettier with some Kotlin magic
```suggestion
        placedMarkers[utility]?.let { markersList ->
            markersList.map {it.remove()}
        }

```